win/sassc.vcxproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <SASSC_VERSION>[NA]</SASSC_VERSION>
     <LIBSASS_VERSION>[NA]</LIBSASS_VERSION>
-    <LIBSASS_DIR>..\..</LIBSASS_DIR>
+    <LIBSASS_DIR Condition="'$(LIBSASS_DIR)' == ''">..\..</LIBSASS_DIR>
```

Comment:
Yup, the default order is:

* property defined by command line arg has highest precedence.
* property defined by environment variable has lowest precedence.
* property defined in `vcxproj` can override environment variable.

This change makes that defined in `vcxproj` the lowest precedence; rendering environment variable being able to override it, while keeping the CLI arg highest precedence.

This is used in AppVeyor.yml in its current form. Otherwise, we will need to change `build` to `build_script` in AppVeyor and manually invoke the `msbuild /p:LIBSASS_DIR win/sassc.sln` command.